include/components/config_parser.hpp

```diff
@@ -0,0 +1,281 @@
+#pragma once
+
+#include <algorithm>
+#include <set>
+
+#include "common.hpp"
+#include "errors.hpp"
+#include "components/logger.hpp"
+#include "components/config.hpp"
+#include "utils/file.hpp"
+#include "utils/string.hpp"
+
+POLYBAR_NS
+
+/**
+ * \brief Generic exception for the config_parser
```

Comment:
Oh. I used `\brief` because the doxygen documentation uses it, but `@brief` would also be accepted by doxygen. I think `\brief` looks nicer what do you think?

Reply:
I don't have a preference, but if we picked `\brief` we'd have to go through the current code and change it. Not a big deal, but it is possible we forget to do this.